pages/docs/couple-your-code/couple-your-code-porting-v2-3.md

```diff
@@ -16,6 +16,91 @@ Please add breaking changes here when merged to the `develop` branch.
 
 ## preCICE API
 
+<!-- Split code block. See https://github.com/precice/precice.github.io/commit/74e377cece4a221e00b5c56b1db3942ec70a6272. -->
```

Comment:
@MakisH Do you have more details on this point? If it is simple: Let's do it now. If it is more complicated: I guess it's not critical and we can just merge.

Reply:
I do not understand what you mean. Could you please elaborate what the problem is? I vaguely remember some issue with code highlighting in diff. Normal diff should work.

Reply:
According to https://github.com/precice/precice.github.io/commit/74e377cece4a221e00b5c56b1db3942ec70a6272 there seems to be some issue with longer code blocks. But let's not worry about it here. I'll open an issue, because we should probably discuss this with @chlorenz and add some general information and guidelines somewhere in the documentation for the documentation.